### PR TITLE
Ignore .log.json

### PR DESCRIPTION
**Pull request by innerlee**
Ignore training log files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+*.log.json
```
Comment (hellock) [inline .gitignore]: maybe also `*.log`

Comment (hellock) [reply .gitignore]: These custom files and folders may be moved to the bottom.

Comment (innerlee) [reply .gitignore]: sure

**Comment by lizxchen** [comment]
@hellock  Hi, which is the file carry out ".log.json"？Thank you.  Because i trained the model in previous version.

**Comment by OpenMMLab-Assistant003** [comment]
Hi @innerlee！First of all, we want to express our gratitude for your significant PR in the MMDetection project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤ @innerlee

**Pull request merged**